script/configs/exclude_all_packages_app_wasm.yaml

```diff
@@ -0,0 +1,18 @@
+# This list should be kept as short as possible, and things should remain here
+# only as long as necessary, since in general the goal is for all of the latest
+# versions of packages to be mutually compatible, and compilable with Wasm.
+
+# This is only used for wasm compilation. Once all packages in the repo have
+# been migrated, remove this file and use `exclude_all_packages_app.yaml` instead.
+
+# Packages that aren't migrated yet.
+# https://github.com/flutter/flutter/issues/117022
+- camera
+- webview_flutter
+
+# Dependencies are not migrated yet
+# https://github.com/flutter/flutter/issues/148624
+- google_maps_flutter
+
+# This is a permanent entry, as it should never be a direct app dependency.
```

Comment:
Rather than require duplicate entries, it would be better to pass both exclusion files in wasm mode, otherwise any change unrelated to wasm would need to be made in both files.

Reply:
Good call, passed one (or two) yaml exclusion files to the script, instead of having duplicate entries.